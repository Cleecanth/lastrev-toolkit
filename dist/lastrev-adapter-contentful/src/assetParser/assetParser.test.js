"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var assetParser_1 = __importDefault(require("./assetParser"));
var assetParser_mock_1 = __importDefault(require("./assetParser.mock"));
describe('assetParser.js', function () {
    test('correctly parsers asset', function () {
        var parsed = assetParser_1.default(assetParser_mock_1.default);
        expect(parsed).toMatchSnapshot();
    });
    test('removes undefined values', function () {
        var _a = assetParser_mock_1.default.fields, title = _a.title, file = _a.file, sys = assetParser_mock_1.default.sys;
        var parsed = assetParser_1.default({
            fields: {
                title: title,
                file: file
            },
            sys: sys
        });
        expect(parsed).toMatchSnapshot();
    });
});
