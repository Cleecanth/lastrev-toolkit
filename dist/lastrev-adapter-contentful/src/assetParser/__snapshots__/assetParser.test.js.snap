// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`assetParser.js correctly parsers asset 1`] = `
Object {
  "contentType": "image/png",
  "description": "A Nyan Cat",
  "filename": "Nyan_cat_250px_frame.png",
  "height": 250,
  "size": 12273,
  "title": "Nyan Cat",
  "url": "//images.ctfassets.net/yadj1kx9rmg0/4gp6taAwW4CmSgumq2ekUm/9da0cd1936871b8d72343e895a00d611/Nyan_cat_250px_frame.png",
  "width": 250,
}
`;

exports[`assetParser.js removes undefined values 1`] = `
Object {
  "contentType": "image/png",
  "filename": "Nyan_cat_250px_frame.png",
  "height": 250,
  "size": 12273,
  "title": "Nyan Cat",
  "url": "//images.ctfassets.net/yadj1kx9rmg0/4gp6taAwW4CmSgumq2ekUm/9da0cd1936871b8d72343e895a00d611/Nyan_cat_250px_frame.png",
  "width": 250,
}
`;
